samples/nrf9160/modem_shell/src/utils/mosh_print.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
+
+#include <zephyr.h>
+#include <posix/time.h>
+#include <shell/shell.h>
+#include <shell/shell_uart.h>
+
+#include "mosh_print.h"
+
+const struct shell *shell_global;
```

Comment:
Is `shell_global` still used outside this module or could it be made `static` and renamed f.ex. to just `shell`?

Reply:
Renamed to `mosh_shell`. I don't particularly like to use `shell `as a name because it could theoretically be overriding a name from zephyr shell includes.